Makefile

```diff
@@ -106,12 +106,6 @@ ci-validate:
 		--pure \
 		--run './$(SCRIPTS_DIR)/validate-packages.sh'
 
-.PHONY: ci-docs
-ci-docs: $(DAML_SDK_ROOT)
-	@nix-shell \
-		--pure \
-		--run 'pipenv run make doc-html'
```

Comment:
It should be enough to leave this target and make it run `pipenv run make doc-code`

Reply:
`doc-code` shouldn't require python as its calling the daml cli?

Reply:
that's right, my bad 😅